warehouse/scripts/templates/ci_report.md

```diff
@@ -0,0 +1,47 @@
+Warehouse report 📦
+
+{% if new_models or changed_incremental_models %}
+### Checks/potential follow-ups
+
+Checks indicate the following action items may be necessary.
+{% if new_models -%}
+- [ ] For new models, do they all have a surrogate primary key that is tested to be not-null and unique?
+{%- endif %}
+{% if changed_incremental_models -%}
+- [ ] For changed incremental models, does the PR description identify whether a full refresh is needed for these tables?
```

Comment:
`changed or downstream`

Reply:
good catch, I need to change the code for this